types/generate-ctrl-c-event/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for generate-ctrl-c-event 1.2
+// Project: https://github.com/zenflow/generate-ctrl-c-event
+// Definitions by: Matthew Francis Brunetti <https://github.com/zenflow>
+//                 Khairul Azhar Kasmiran <https://github.com/kazarmy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace GenerateCtrlCEvent {
+  interface Funcs {
```

Comment:
@kazarmy Just curious - Why not just `declare interface GenerateCtrlCEvent {` (skipping the namespace and giving a better name to the interface)?

Reply:
Yes, using just an `interface` is better (ef1720d). I added the namespace because I thought it solved the following problem (under VSCode 1.55.0 with Remote-WSL 0.54.6):

![generateCtrlC-any-2](https://user-images.githubusercontent.com/12002672/113468691-5a7d1580-947a-11eb-86b3-53725415c724.png)

But after further testing today, I realize that I just need to be more patient and stop checking the type of `generateCtrlC` as VSCode boots up.